front/src/utils/__tests__/geometry.test.ts

```diff
@@ -71,3 +83,26 @@ describe('getCurrentBearing', () => {
     expect(result).toBeCloseTo(expected, 2);
   });
 });
+
+describe('nearestPointOnLine', () => {
+  it('should works', () => {
+    [
+      linePointOnLeft,
+      linePointOnRight,
+      lineNorthenLatitude,
+      linePointOnVertex,
+      linePointOnVertexUp,
+      line1,
+      route1,
+      route2,
+      smallInfra,
```

Comment:
What does smallInfra test specifically ? I have the feeling it does not bring anything new, but I may be wrong

Reply:
It's the original test for the linked issue, the one that allows me to reproduce  the issue.